### PR TITLE
New versioned architecture wrappers GTM 3.0 and 3.1

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

**Comment by victorwiebe** [comment]
Take close note of my changes in usdTieredSTO

File: src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/common.ts [hunk 1]
```diff
@@ -26,7 +20,7 @@ import {
   weiToValue,
 } from '../../../../utils/convert';
 import functionsUtils from '../../../../utils/functions_utils';
-import GeneralTransferManagerWrapper from '../../transfer_manager/general_transfer_manager_wrapper';
+import { GeneralTransferManager_3_0_0 } from '../../transfer_manager/general_transfer_manager_wrapper';
```
Comment (monitz87) [inline src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/common.ts]: This should be the common GTM wrapper

File: src/contract_wrappers/modules/transfer_manager/general_transfer_manager_wrapper/__tests__/3.0.0.test.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { mock, instance, reset } from 'ts-mockito';
+import {
+  GeneralTransferManagerContract_3_0_0,
+  ISecurityTokenContract_3_0_0,
+  PolyTokenEvents_3_0_0,
+  BigNumber,
+  Web3Wrapper,
+} from '@polymathnetwork/abi-wrappers';
+import ContractFactory from '../../../../../factories/contractFactory';
+import {GeneralTransferManager_3_0_0} from '../3.0.0';
```
Comment (monitz87) [inline src/contract_wrappers/modules/transfer_manager/general_transfer_manager_wrapper/__tests__/3.0.0.test.ts]: Should have spaces between the brackets and the class name

File: src/contract_wrappers/modules/transfer_manager/general_transfer_manager_wrapper/__tests__/3.1.0.test.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
+// GeneralTransferManager test
```
Comment (monitz87) [inline src/contract_wrappers/modules/transfer_manager/general_transfer_manager_wrapper/__tests__/3.1.0.test.ts]: You forgot to test the new `whitelistModule` function

File: src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/common.ts (hunk 1 above)
Comment (victorwiebe) [reply src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/common.ts]: I'm confused about this. I tried using common in place of 3.0.0 and it didn't work.

**Pull request merged**